### PR TITLE
Wincrate

### PR DESCRIPTION
**Pull request by qrnch-jan**
Use the windows-sys crate rather than winapi.  Right now we're in the unfortunate situation where building Windows applications in Rust tends to eventually pull in multiple Windows API wrappers.  It _seems_ like the current trend is to convert projects to use windows{-sys} (since it's actively developed and supported by Microsoft).

### CODE REVIEW

**Review by Stargateur — APPROVED** [review]
why not thus the doc melt rustdoc https://docs.rs/windows-sys

But the last version of windows-sys doesn't fit the current pcap MSRV https://github.com/microsoft/windows-rs/blob/master/crates/libs/sys/Cargo.toml#L11

**Comment by qrnch-jan** [comment]
Is the MSRV issue merely a documentation issue?  Should it be noted in `README.md` that Windows, without `capture-stream`, builds require `>= 1.46.0`?

**Comment by qrnch-jan** [comment]
I updated the changelog and the readme, but I have never used github workflows before, so I'm fairly certain I borked something.

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +22,7 @@
 ### Changed
 
 - `capture-stream` requires rustc version 1.49.0 due to dependency on `tokio`.
+>>>>>>> master
```
Comment (Wojtek242) [inline CHANGELOG.md:25]: Accidentally left over, please remove.

File: README.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ The feature `capture-stream` depends on `tokio`, but we only lock `tokio` versio
 
 ## Windows
 
+As of 0.9.3 Windows builds, without the `cature-stream` feature, requires a compiler version >= 1.46.0.
```
Comment (Wojtek242) [inline README.md:32]: Next release will `0.10.0`

**Review by Wojtek242 — CHANGES_REQUESTED** [review]
Correct me if I'm wrong, but Windows doesn't really have something like Linux distributions where a particular package may be locked to a specific version in an LTS release for many years. In which case if `windows-sys` is really much better then it's okay having a different MSRV for Windows.

Anyway, some very minor changes and it can be merged.

File: CHANGELOG.md (hunk 1 above)
Comment (sekineh) [reply CHANGELOG.md]: @Wojtek242 , did you mean to remove line 25 only? Or to remove both line 24 and 25?

**Review by Wojtek242 — APPROVED** [review]

**Comment by Wojtek242** [comment]
Merged manually due to merge conflicts with sendqueue

**Pull request closed without merging**